docs/src/main/sphinx/connector/hive.rst

```diff
@@ -649,381 +432,412 @@ Property Name                              Description
+``partition_projection_ignore`` table property to ``true`` for a table to bypass
+any errors.
 
-``hive.metastore.glue.aws-access-key``               AWS access key to use to connect to the Glue Catalog. If
-                                                     specified along with ``hive.metastore.glue.aws-secret-key``,
-                                                     this parameter takes precedence over
-                                                     ``hive.metastore.glue.iam-role``.
+Refer to :ref:`hive_table_properties` and :ref:`hive_column_properties` for
+configuration of partition projection.
 
-``hive.metastore.glue.aws-secret-key``               AWS secret key to use to connect to the Glue Catalog. If
-                                                     specified along with ``hive.metastore.glue.aws-access-key``,
-                                                     this parameter takes precedence over
-                                                     ``hive.metastore.glue.iam-role``.
+Metastore configuration for Avro
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``hive.metastore.glue.catalogid``                    The ID of the Glue Catalog in which the metadata database
-                                                     resides.
+In order to enable first-class support for Avro tables when using
+Hive 3.x, you must add the following property definition to the Hive metastore
+configuration file ``hive-site.xml`` and restart the metastore service:
 
-``hive.metastore.glue.iam-role``                     ARN of an IAM role to assume when connecting to the Glue
-                                                     Catalog.
+.. code-block:: xml
 
-``hive.metastore.glue.external-id``                  External ID for the IAM role trust policy when connecting
-                                                     to the Glue Catalog.
+   <property>
+        <!-- https://community.hortonworks.com/content/supportkb/247055/errorjavalangunsupportedoperationexception-storage.html -->
+        <name>metastore.storage.schema.reader.impl</name>
+        <value>org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader</value>
+    </property>
 
-``hive.metastore.glue.partitions-segments``          Number of segments for partitioned Glue tables, defaults
-                                                     to ``5``.
+Storage
+-------
 
-``hive.metastore.glue.get-partition-threads``        Number of threads for parallel partition fetches from Glue,
-                                                     defaults to ``20``.
+The Hive connector supports the following storage options:
 
-``hive.metastore.glue.read-statistics-threads``      Number of threads for parallel statistic fetches from Glue,
-                                                     defaults to ``5``.
+* :doc:`Amazon S3 <hive-s3>`
+* :doc:`Azure Storage <hive-azure>`
+* Google Cloud Storage :ref:`properties <hive-google-cloud-storage-configuration>`
```

Comment:
Super nitpick, but the phrasing of this list is a little off to me, maybe move properties and configuration to sub-bullets? This just feels more consistent to me, but feel free to ignore

- Amazon S3
- Azure ABFS and WASB
- Google Cloud Storage
  - Properties
  - Configuration
- IBM Cloud Object Storage